runner/analyze.py

```diff
@@ -207,8 +180,12 @@ def _perform_reconstruction_analysis(
     ]
 
     for decoder in reconstruction_decoders:
-        rec_dict = reconstruct_images(device, brain, decoder, train_set, test_set, 5)
-        recon_fig = plot_reconstructions(**rec_dict, num_samples=5)
+        (norm_means, norm_stds), rec_dict = reconstruct_images(
```

Comment:
I think it would be better to 'unpack' the normalization parameters here directly from the train set (instead of inside reconstruct images:
```
(norm_means, norm_stds) = train_set.normalization_stats
rec_dict = reconstruct_images(...) #change this method interface back to returning only the dict
```

Reply:
yes indeed